templates/cassiopeia/scss/vendor/choicesjs/choices.scss

```diff
@@ -1,14 +1,13 @@
 @import "../../../../../media/vendor/bootstrap/scss/functions";
 
 // Cassiopea Variables, Functions and Mixins
```

Comment:
Does this comment belong to the moved code below it? If so, shouldn't it be moved, too?

Reply:
With moved code I mean the `@import "../../tools/tools";` line.

Reply:
@drmenzelit Why did you mark this as resolved? Do you really think the comment line `// Cassiopea Variables, Functions and Mixins` fits to the loading of the bootstrap stuff which now comes below it after this PR has moved the `@import "../../tools/tools";` to below the bootstrap stuff?